pkgs/top-level/python-packages.nix

```
@@ -905,6 +905,12 @@ self: super: with self; {
 
   arris-tg2492lg = callPackage ../development/python-modules/arris-tg2492lg { };
 
+  arro3-compute = (callPackage ../development/python-modules/arro3 { }).arro3-compute;
+
+  arro3-core = (callPackage ../development/python-modules/arro3 { }).arro3-core;
+
+  arro3-io = (callPackage ../development/python-modules/arro3 { }).arro3-io;
+
```

1. When calling a set of packages (rather than a package) you should use `callPackages` not `callPackage`
   This will add `.override` to each package in the set, rather than to the set itself.
2. Here, it'd be simpler to inherit each package from a single call:

```suggestion
  inherit (callPackages ../development/python-modules/arro3 { })
    arro3-compute
    arro3-core
    arro3-io
    ;

```

---

I'm not aware of callPackages, and see I cannot access it in `python-packages.nix`. I'm not really sure how to correctly import this without the risk of breaking anything, can you help me here?  

In addition we would loose the nice sort by alphabet, as the line starts with inherit. Not sure how much that would matter though.